python/mxnet/optimizer.py

```diff
@@ -174,14 +174,24 @@ def set_lr_scale(self, args_lrscale): # pylint: disable=unused-argument
         raise DeprecationWarning
 
     def set_lr_mult(self, args_lr_mult):
-        """Set individual learning rate for each weight.
+        """Set an individual learning rate multiplier for each parameter.
+
+        If you specify a learning rate multiplier for a parameter, then
+        the learning rate for the parameter will be set as the product of
+        the global learning rate `self.lr` and its multiplier.
+
+        .. note:: The default learning rate multiplier of a `Symbol`
```

Comment:
https://github.com/dmlc/mxnet/pull/6120/commits/7a769fa49ca540fc202056d7aab1eb156fdb2d88#diff-c445cb799a50a6e48c48d19a1ef37d3bL191 uses ``__lr_mult__`` attribute, so that's what I wanted to mention here. Do you think my description is incorrect?